scripts/messages/message_template.msg

```diff
@@ -9144,7 +9144,7 @@ version 2.0
 // message will be resent a few times in the hopes the server finally receives it.
 //
 {
-	GameControlInput Medium 431 NotTrusted Zerocoded
+	GameControlInput High 32 NotTrusted Zerocoded
```

Comment:
Thanks for making these `message_template.msg` changes.  I will be cutting a corresponding PR on the **server** code later today.